From 885c5c7a71dc2b377f8df5dc926fc2d5b3ff9cf1 Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Tue, 16 Apr 2024 11:23:39 +0200
Subject: [PATCH 45/68] ustream-ssl: update to Git HEAD (2024-04-07)

91666a38b7bd ustream-mbedtls: Add compatibility with Mbed TLS 3.0.0
263b9a97cf7e cmake: Fail if undefined symbols are used
7621339d7694 mbedtls: fix build on non-linux systems
268050964b08 ustream-mbedtls: add missing psa_crypto_init call
956fba242ac0 add callbacks for debug messages
9fdf3fb87af5 mbedtls: add TLS 1.3 ciphers
28c4c1e6471b mbedtls: disable TLS 1.3 in client mode when skipping verification
d61493a44204 mbedtls: add missing ifdef for build with disabled debug

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---
 package/libs/ustream-ssl/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/package/libs/ustream-ssl/Makefile b/package/libs/ustream-ssl/Makefile
index ae5c8026c2..c042215105 100644
--- a/package/libs/ustream-ssl/Makefile
+++ b/package/libs/ustream-ssl/Makefile
@@ -5,9 +5,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/ustream-ssl.git
-PKG_SOURCE_DATE:=2023-02-25
-PKG_SOURCE_VERSION:=498f6e268d4d2b0ad33b430f4ba1abe397d31496
-PKG_MIRROR_HASH:=a201d065dd613e30886c9f13a0851cec79538192cfe591b6f8ecd88724d55fb1
+PKG_SOURCE_DATE:=2024-04-07
+PKG_SOURCE_VERSION:=d61493a4420417cbf9931ffee8c862faf04f2967
+PKG_MIRROR_HASH:=b55f280c6203d14f589a8a6c89ebc6b059af99138e972aa8f5992ce71e891844
 CMAKE_INSTALL:=1
 
 PKG_LICENSE:=ISC
-- 
2.34.1

